.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
```

Comment:
Have you tested this workflow?

Reply:
I'll do it now

Reply:
Hi, Frank, I have tested the workflow, it works well.